docker/server/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# NOTE: The docker build context directory must be the root of the source tree.
+# NOTE: To build the release image, SQLFlow must be built into ./build
+FROM ubuntu:18.04
+
+# Install and configure the pre-built SQLFlow binaries.
+COPY build /build
+ENV SQLFLOW_PARSER_SERVER_PORT=12300
+ENV SQLFLOW_PARSER_SERVER_LOADING_PATH="/usr/local/sqlflow/java"
+
```

Comment:
`python/sqlflow_submitter` folder is also needed.

``` docker
COPY python /usr/local/sqlflow/python
ENV PYTHONPATH=/usr/local/sqlflow/python:$PYTHONPATH
```

Reply:
I assume these should be in the step image, other than the server image, am I right?

Reply:
I moved them into the server image, as I think they are necessary for the server to all Fluid/Couler, right?

Reply:
Yes, that's right.